ios/MullvadVPN/AppDelegate.swift

```diff
@@ -437,6 +439,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         )
     }
 
+    private func checkWipeSettings() {
+        guard var previousAppVersion = try? SettingsManager.getPreviousAppVersion() else { return }
```

Comment:
Looks like it can be a constant.

```suggestion
        guard let previousAppVersion = try? SettingsManager.getPreviousAppVersion() else { return }
```